README.md

```diff
@@ -1,2 +1,37 @@
 # storcycle_api
-StorCycle API Examples
+
+[Swagger Documentation](https://localhost/apidocs/)
+
+## StorCycle API Examples
+
+This repository contains examples for interacting with the StorCycle API. These general examples require either creation of the example directory tree, or adjusting variables to suit the local system.
+
+To use the examples, client code must first be generated in the desired language.
+
+### Python:
```

Comment:
I'd put in a command line example for doing the generation also.  `java -jar openapi/codegen/1283369100_openapi-generator-cli-4.3.1.jar generate -g python -i openapi.yaml -o /tmp/`

Reply:
I assume I should also provide the link to [https://openapi-generator.tech/docs/installation/](https://openapi-generator.tech/docs/installation/) to get the client generator?

Reply:
Sounds good